v2/config/samples/network/v1beta/v1beta20180901_privatednszone.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: network.azure.com/v1beta20180601
+kind: PrivateDnsZone
+metadata:
+  creationTimestamp: null
```

Comment:
Do we require this and empty status below?

Reply:
Not required; will remove.